source/brailleDisplayDrivers/albatross/readme.md

```diff
@@ -0,0 +1,24 @@
+## Albatross
+
+This module contains driver for Caiku Albatross 46 and 80 braille displays.

```

Comment:
The PR description you have written is really helpful and informative.
Could you format the outline you wrote up into this readme?